<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Select Elements By Layer By GTVFX</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="listWidget_layers">
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="alternatingRowColors">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Remove Existhing Light Select Elements</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_create">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>50</height>
       </size>
      </property>
      <property name="text">
       <string>Create Selected Elements</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>367</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGTVFX">
    <property name="title">
     <string>GTVFX</string>
    </property>
    <addaction name="act_help"/>
    <addaction name="act_gtvfx"/>
   </widget>
   <addaction name="menuGTVFX"/>
  </widget>
  <action name="act_help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="act_gtvfx">
   <property name="text">
    <string>www.gtvfx.com</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
